src/main/xerus/monstercat/api/Player.kt

```diff
@@ -142,6 +148,10 @@ object Player: FadingHBox(true, targetHeight = 25) {
 			play()
 			setOnReady {
 				label.text = "Now Playing: $track"
+				if(Files.exists(Settings.PLAYEREXPORTFILE()) && !Files.isDirectory(Settings.PLAYEREXPORTFILE())) {
```

Comment:
why does the file have to exist? shouldn't it be created if it doesn't exist?

Reply:
Yeah it should, I'll try to find it.

Reply:
926341eadfcab0c3fca4f85fc144245282ebe450